test/karma/karma.conf.js

```diff
@@ -73,6 +73,7 @@ module.exports = function( config ) {
 		colors: !isTravis,
 
 		port: 9876,
+		hostname: "bs-local.com",
```

Comment:
Where does that hostname come from?

Reply:
It is needed to run tests on physical iOS devices as those devices don't support forwarding request for loopback addresses to the proxy. Also, `bs-local.com` points to 127.0.0.1. So it shouldn't create an issue while running tests anywhere.

Reply:
Nice, I didn't know about this! Could you add a comment above this line to explain it?

Reply:
One problem with it - this hostname won't work when tests are run locally, will it? Perhaps we should set it to `"bs-local.com"` only on Travis and only when BrowserStack is used (i.e. not in PRs)? I'm actually surprised PhantomJS test runs succeeded on Travis with this setting enabled; do you know why?